Packs/LogPoint_SIEM_Integration/Integrations/LogPoint_SIEM_Integration/LogPoint_SIEM_Integration.py

```diff
@@ -467,6 +588,115 @@ def get_users_command(client):
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    del result['success']
+    table_header = list(result.keys())
+    display_title = "User's Preference"
+    markdown = tableToMarkdown(display_title, result, headers=table_header)
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.User.Preference',
+        outputs=result
+    )
+
+
+def get_logpoints_command(client):
+    result = client.get_logpoints()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    table_header = []
+    display_title = "LogPoints"
+    allowed_loginspects = result.get('allowed_loginspects')
+    if allowed_loginspects and len(allowed_loginspects) > 0:
+        table_header = list(allowed_loginspects[0].keys())
+    markdown = tableToMarkdown(display_title, allowed_loginspects, headers=table_header)
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.LogPoints',
+        outputs_key_field='ip',
+        outputs=allowed_loginspects
+    )
+
+
+def get_repos_command(client):
+    result = client.get_repos()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    table_header = []
+    display_title = "LogPoint Repos"
+    allowed_repos = result.get('allowed_repos')
+    if allowed_repos and len(allowed_repos) > 0:
+        table_header = list(allowed_repos[0].keys())
+    markdown = tableToMarkdown(display_title, allowed_repos, headers=table_header)
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.Repos',
+        outputs_key_field='repo',
+        outputs=allowed_repos
+    )
+
+
+def get_devices_command(client):
+    result = client.get_devices()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    table_header = []
+    display_title = "Devices"
+    allowed_devices = result.get('allowed_devices')
+    if allowed_devices and len(allowed_devices) > 0:
+        table_header = list(allowed_devices[0].keys())
+    markdown = tableToMarkdown(display_title, allowed_devices, headers=table_header)
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.Devices',
+        outputs=allowed_devices
+    )
+
+
+def get_livesearches_command(client):
+    result = client.get_livesearches()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    display_title = "Live Searches"
+    livesearches = result.get('livesearches')
+    markdown = tableToMarkdown(display_title, livesearches, headers=None)
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.LiveSearches',
+        outputs=livesearches
+    )
+
+
+def search_logs_command(client, args):
+    query = args.get('query')
+    time_range = args.get('time_range') if args.get('time_range') else 'Last 5 minutes'
```

Comment:
```suggestion
    time_range = args.get('time_range', 'Last 5 minutes')
```

Reply:
Please note that we have refactored the default values for the arguments of `lp-search-logs` command in this [10d1ffd](https://github.com/demisto/content/pull/12790/commits/10d1ffdc93b25bdca537dfc5cb8e5f3c6fffdb97) commit.